ffs/scheduler/scheduler_storage.go

```diff
@@ -202,7 +202,9 @@ func (s *Scheduler) executeHotStorage(ctx context.Context, curr ffs.CidInfo, cfg
 		return ffs.HotInfo{Enabled: false}, nil
 	}
 
-	sctx, cancel := context.WithTimeout(ctx, time.Second*time.Duration(cfg.Ipfs.AddTimeout))
+	// ToDo: this is a hot-fix to force a big timeout until we have a
+	// migration tool to make this tunable again.
+	sctx, cancel := context.WithTimeout(ctx, time.Second*300)
```

Comment:
See [this](https://textile-public.slack.com/archives/GQRACAG3G/p1600958777011300).